### DISCUSSION TITLE
zsh: command not found: bridgetown

### DISCUSSION THREAD

**Discussion by tomdringer** [bridgetownrb/bridgetown#520] [Q&A]
I've installed the Bridgetown gem running Ru by 3.1.0 but i'm getting 'zsh: command not found: bridgetown' when running the 'bridgetown new' command. I think i have everything correct in my $path (I'm running MacOS).
Any other MacOS users having issues? Thanks

**Comment by daogu** [comment]
`gem env`
Find "EXECUTABLE DIRECTORY"
like EXECUTABLE DIRECTORY: /opt/homebrew/lib/ruby/gems/3.1.0/bin
then add this PATH to .zshrc or /etc/paths ，remember source
